packages/ove-service-layout/src/index.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+// custom validator extensions
+require("./validator/extensions");
+
+let logger = require("debug")("layout:app");
```

Comment:
Propose that we use `const` insteaf of `let` for anything that does not change (especially for require/imports). You may find that this is a fairly consistent practice within other parts of OVE. Please also fix the same in other files.

Reply:
https://mathiasbynens.be/notes/es6-const